docs/geedocs/5.2.1/answer/7160001.html

```diff
@@ -102,119 +207,301 @@ <h5>
-          <td>No current work around</td>
+          <td>No current work around.</td>
         </tr>
+        <tr>
+          <td>442</td>
+          <td>Multiple database pushes after upgrade don't report a warning</td>
+          <td>No current work around.</td>
+        </tr>		
         <tr>
           <td>444</td>
           <td>Fusion installer does not upgrade the asset root on RHEL 7</td>
-          <td>Upgrade the asset root manually by running the command that is printed when you try to start the fusion service</td>
+          <td>Upgrade the asset root manually by running the command that is printed when you try to start the fusion service.</td>
         </tr>
+        <tr>
+          <td>445</td>
+          <td>Path to tutorial source volume in gee_test instructions is different from path used in installers</td>
+          <td>Use <code>/opt/google/share/tutorials</code>.</td>
+        </tr>		
+        <tr>
+          <td>448</td>
+          <td>Investigate Out of Memory issues</td>
+          <td>Use a system that have more than 4GB RAM.</td>
+        </tr>		
         <tr>
           <td>453</td>
           <td>Improve `check_server_processes_running` detection for uninstall</td>
-          <td>No current work around</td>
+          <td>No current work around.</td>
         </tr>
         <tr>
           <td>456</td>
           <td>Inconsistent behavior of vector layers after upgrade</td>
-          <td>No current work around</td>
+          <td>No current work around.</td>
         </tr>
+        <tr>
+          <td>460</td>
+          <td>Possibility of seg fault in QDateWrapper</td>
+          <td>No current work around.</td>
+        </tr>		
         <tr>
           <td>474</td>
           <td>Running gee_check on some supported platforms reports that the platform is not supported</td>
           <td>You can ignore the failed test if using a supported platform (Ubuntu 14.04, Ubuntu 16.04, RHEL 7, and CentOS 7).</td>
         </tr>
+        <tr>
+          <td>477</td>
+          <td>'service geserver stop/start/restart' doesn't work on Ubuntu 16.04 without a reboot</td>
+          <td>Reboot and try again</td>
+        </tr>
+        <tr>
+          <td>487</td>
+          <td>gdal - python utilities do not recognize osgeo module</td>
+          <td>Install <code>python-gdal</code>.</td>
+        </tr>		
+        <tr>
+          <td>507</td>
+          <td>Volume host is reported unavailable if `hostname` doesn't match volume host</td>
+          <td>Set the host values in <code>/gevol/assets/.config/volumes.xml</code> to the FQDN and restart the fusion service.</td>
+        </tr>		
+        <tr>
+          <td>535</td>
+          <td>DownloadTutorial.sh often is not staged properly for install</td>
+          <td>Copy <code>DownloadTutorial.sh</code> to <code>/tmp/fusion_os_install</code>.</td>
+        </tr>		
+        <tr>
+          <td>557</td>
+          <td>Wms service problem with 'width' & 'height' & 'bbox'</td>
```

Comment:
Shouldn't these be escaped? `&amp;` instead of `&`.

Reply:
done